Xamarin.PropertyEditing.Mac/Controls/Custom/NumericSpinEditor.cs

```diff
@@ -13,24 +13,25 @@ internal class NumericSpinEditor<T> : NumericSpinEditor
 
 	internal class NumericSpinEditor : NSView, INSAccessibilityGroup
 	{
-		const int stepperSpace = 2;
-		const int stepperWidth = 11;
-		const int stepperTopHeight = 9;
-		const int stepperBotHeight = 10;
+		internal const int stepperSpace = 2;
```

Comment:
Why are we making these internal?

Reply:
I didn't think we wanted to expose them, but I'll change it back.